command/pki.go

```diff
@@ -34,5 +35,49 @@ Usage: vault pki <subcommand> [options] [args]
 }
 
 func (c *PKICommand) Run(args []string) int {
+	//c.testCreateRoot()
```

Comment:
I don't think this belongs here actually; this would belong in a new file and be added as a `pki add-intermediate` or `pki add-root` type behavior in the `command/commands.go` file.

Reply:
Yes, this was just an easy place for me go invoke test code.